tools/sesman/sesmanworker/engine.py

```diff
@@ -614,10 +618,31 @@ def notify_find_process_rdp(self, regex, deny, app_name, app_cmd_line,
             Logger().info("Engine NotifyFindProcessInRDPFlow failed: "
                           "(((%s)))" % (traceback.format_exc()))
 
+    def _get_target_filter_mode(self, device_filter):
```

Comment:
Move this function in targetaccuratefilter module.
rename it to `get_rdp_selector_filter_mode(self, group_filter, device_filter, protocol_filter)`